Basic/DyanmicPrefabs/Assets/Scripts/ConnectionsUI.cs

```diff
@@ -0,0 +1,325 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UIElements;
+
+public class ConnectionsUI : MonoBehaviour
```

Comment:
Just a note to namespace this and the other script. How about Game.UI? Oh and yeah they can go in their own folder too.